index.html

```diff
@@ -0,0 +1,344 @@
+<!DOCTYPE html>
```

Comment:
## Improve the page layout 

- [ ] Since we are `building with responsive design`, we need to make sure nothing is messed up. The first three articles of the page layout are the only ones that get to span the entire page width the rest should occupy a much smaller width and their contents confined and centred in the middle of the view width. Would you mind effecting these changes to make our page resemble the original page with the contents of this milestone as an inspiration? Below I have tried to share the comparisons. 
- [ ] The fonts of the clone page also need to be much bolder and of the exact font type.
- [ ] Images are also not loading. let's work on them as well. 
- [ ] Take care of the footer as well. We do have some unnecessary spacing at the bottom.
- [ ] Do you mind validating the markup to remove the current earnings that we have? You can use this [tool](https://validator.w3.org/nu/#textarea).
- [ ] Have a look at the comparisons below 👇 

### Layout 
|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/verissimohenry/TNW-Prema/mileston1-mainsection/index.html)|
|:--:|:--:|
|![tnw](https://user-images.githubusercontent.com/22345961/109838254-0bfc1180-7c57-11eb-8c21-d88c364d5543.png)|![tnw-clone](https://user-images.githubusercontent.com/22345961/109838308-174f3d00-7c57-11eb-8ccc-0a2d55f6ccdf.png)|

### Footer

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/verissimohenry/TNW-Prema/mileston1-mainsection/index.html)|
|:--:|:--:|
|![footer](https://user-images.githubusercontent.com/22345961/109839536-3ef2d500-7c58-11eb-9f8b-3f174fd59e4d.png)|![footer-clone](https://user-images.githubusercontent.com/22345961/109839555-431ef280-7c58-11eb-8583-ae3447ca0ae4.png)|